backend/services/mgmt/v1alpha1/job-service/runs.go

```diff
@@ -273,6 +273,10 @@ func (s *Service) getEventsByWorkflowId(
 			isRunComplete = true
 		case enums.EVENT_TYPE_WORKFLOW_EXECUTION_TIMED_OUT:
 			isRunComplete = true
+		case enums.EVENT_TYPE_WORKFLOW_EXECUTION_TERMINATED:
```

Comment:
these were causing the run events to still be refreshed even though the job was complete